app/boards/arm/nrf52840_m2/nrf52840_m2.dts

```diff
@@ -51,16 +51,6 @@
 	status = "okay";
 };
 
-&uart0 {
-	compatible = "nordic,nrf-uarte";
-	status = "okay";
-	current-speed = <115200>;
-	tx-pin = <19>;
-	rx-pin = <21>;
-	rts-pin = <23>;
-	cts-pin = <25>;
-};
```

Comment:
According to the makerdiary docs / stock dts file (apache 2.0 licensed) the nrf52840 has a uart and has the following definition per the rest of this PR.

```
&uart0 {
	compatible = "nordic,nrf-uart";
	current-speed = <115200>;
	tx-pin = <16>;
	rx-pin = <15>;
	//rts-pin = <14>;
	//cts-pin = <13>;
};
```

Reply:
Oh. I see that when plugged into https://wiki.makerdiary.com/nrf52840-m2-devkit/ dev kit, those are the pins they use. That seems to me to be something you'd add to the devkit definition, not to this. one.

Reply:
Agreed, closing review / comment.